packages/ui-components/src/components/common/VideoWrapper.stories.ts

```diff
@@ -0,0 +1,61 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
+import CommonVideoWrapper from '~~/src/components/common/VideoWrapper.vue'
+
+type StoryType = StoryObj<{
+  autoplay: boolean
+  muted: boolean
+  controls: boolean
+  vimeoId: string
+}>
+
+export default {
+  title: 'Components/Common/VideoWrapper',
+  component: CommonVideoWrapper,
+  argTypes: {
+    autoplay: {
+      control: { type: 'boolean' },
+      defaultValue: false
```

Comment:
defaultValue is redundant if you set up all args in the Default story (and use that as a base for all other stories)